app/Models/User.php

```diff
@@ -189,6 +193,12 @@ public function suspend()
      */
     public function unSuspend()
     {
+        foreach ($this->servers as $server){
+          if ($this->credits >= $server->product->getHourlyPrice()) {
+            $server->unSuspend();
+        }
```

Comment:
Make sure the formatting is correct when pushing a PR to keep things clean and have the brackets aligned.
Seems like you need to select line 198 and 199 and press tab and it should be fixed.

Reply:
Got fixed in the new commit 👍